internal/mode/static/health_test.go

```diff
@@ -0,0 +1,16 @@
+package static
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("health", func() {
```

Comment:
Think this should use normal Go test instead of the framework?

Reply:
Yeah, since we aren't testing an exported interface we can use Go test.